mailphax.rb

```diff
@@ -2,62 +2,170 @@
 require 'phaxio'
 require 'mail'
 require 'pony'
+require 'tempfile'
+require 'openssl'
+require 'to_regexp'
 
-if not ENV['PHAXIO_KEY'] or not ENV['PHAXIO_SECRET']
-  raise "You must specify your phaxio API keys in PHAXIO_KEY and PHAXIO_SECRET"
+if not ENV['PHAXIO_KEY'] or not ENV['PHAXIO_SECRET'] or not ENV['MAILGUN_KEY']
+  raise "You must specify the required environment variables"
 end
 
 get '/' do
-  "Mailfax v1.0 - Visit a mail endpoint: (/sendgrid, /mandrill, /mailgun)"
+  "MailPhax v1.0 - Visit a mail endpoint: (/mailgun)"
 end
 
+get '/mailgun' do
+  [400, "Mailgun supported, but callbacks must be POSTs"]
+end
 
-get '/mandrill' do
-  [501, "mandrill not implemented yet"]
+$recipientWhitelist = nil
+
+def getRecipientWhitelist()
+  if $recipientWhitelist.nil?
```

Comment:
I think they are useful, and so I've added to the pull request, but I can also see it being too granular. If it is a worthwhile feature, I can add some documentation.